internal/policies/mount/internal_test.go

```diff
@@ -125,6 +125,46 @@ func TestWriteFileWithUIDGID(t *testing.T) {
 	}
 }
 
+func TestCreateUnits(t *testing.T) {
+	t.Parallel()
+
+	tests := map[string]struct {
+		entry   string
+		wantErr bool
+	}{
+		"write single unit":                         {entry: "entry with one value"},
+		"write multiple units":                      {entry: "entry with multiple values"},
+		"write anonymous unit with correct options": {entry: "entry with one anonymous value"},
+
+		"error when writing bad formatted unit":                             {entry: "entry with bad value", wantErr: true},
+		"error when writing multiple units with badly formatted among them": {entry: "entry with correct and bad values", wantErr: true},
```

Comment:
This is supposed to test that, even though there were failures during the process, the "good" units were written correctly. So I need to see if an error was emitted (because there were failures during the process) whilst also checking the correct units...

Reply:
Yes. This was before the change to define where it should or shouldn't fail. So right now, we don't care about this test because if something fails mid process, login will be prevented and then it doesn't matter if the good ones were written perfectly.